### DISCUSSION TITLE
Add a button to insert screenshot from clipboard (I know it works with Ctrl+V))

### DISCUSSION THREAD

**Discussion by Isolator70** [TenderOwl/Frog#162] [Ideas]
I guess many users are not aware of the Ctrl+V capability of Frog. Thinking about the Linux Mint users (like me) which are unable to use the "Take a Screenshot button", it could improve Frog's usability to have also a "Insert picture from Clipboard" button.

**Comment by amka** [comment]
@Isolator70  Hey! There is a button!

![image](https://github.com/TenderOwl/Frog/assets/519146/24c26923-d336-4047-b791-3f1b10476abb)

**Comment by Isolator70** [comment]
Oh, I see 🙈 - thanks. Then it could only be discussed if this button is located prominent enough, otherwise this topic could be closed...

**Comment by amka** [comment]
@Isolator70   If you have any ideas - tell me! I have some ideas and testing them out, but none of them impressed me yet (I mean 3 vertical or horizontal buttons or even a kinda way of "teaching users"). Still thinking...

**Comment by Isolator70** [comment]
> ....any ideas - tell me...

Click the frog - which does a funny animation with it's tongue - to grab clipboard's content. Hover the frog (which changes it's style while mouseover to make clear it is an active element) to get the tip what it will do when clicked.

**Comment by amka** [comment]
I'll add a button to the welcome page.
